charts/mlrun-ce/requirements.lock

```diff
@@ -1,10 +1,10 @@
 dependencies:
```

Comment:
Align file with requirements.yml. You can do this by running `helm dependency update`